plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/AppEngineStandardDeployCommandHandler.java

```diff
@@ -0,0 +1,24 @@
+package com.google.cloud.tools.eclipse.appengine.deploy;
+
+import org.eclipse.core.commands.AbstractHandler;
+import org.eclipse.core.commands.ExecutionEvent;
+import org.eclipse.core.commands.ExecutionException;
+import org.eclipse.jface.dialogs.MessageDialog;
+import org.eclipse.ui.handlers.HandlerUtil;
+
+public class AppEngineStandardDeployCommandHandler extends AbstractHandler {
+
+  @Override
+  public Object execute(ExecutionEvent event) throws ExecutionException {
```

Comment:
I'm using here the new AbstractHandler from the org.eclipse.core.commands bundle as instructed by the deprecation message.
Looking at how classes in the platform implement handlers, `execute()` is still the method to use. `fireHandlerChanged` is used when the handler changes, e.g. `isEnabled()` changes, or some other (not yet existing) property of the handler.